vembrane/representations.py

```diff
@@ -184,7 +194,29 @@ def __init__(
         # REF/ALT alleles are cached separately to raise "MoreThanOneAltAllele"
         # only if ALT (but not REF) is accessed (and ALT has multiple entries).
         self._alleles = None
-        self._func = eval(f"lambda: {expression}", self, {})
+
+        func = f"lambda: {expression}"
+
+        # The VCF specification only allows 32bit floats.
+        # Comparisons such as `INFO["some_float"] > CONST` may yield unexpected results,
+        # because `some_float` is a 32bit float while `CONST` is a python 64bit float.
+        # Example: `c_float(0.6) = 0.6000000238418579 > 0.6`.
+        # We can work around this by wrapping user provided floats in numpy.float32
+        # as follows:
```

Comment:
This text needs to be adjusted to the newest solution:
```suggestion
        # We can work around this by wrapping user provided floats in `ctypes.c_float`,
        # which should follow the same IEEE 754 specification as the VCF spec, as most
        # C compilers should follow this standard (https://stackoverflow.com/a/17904539):
```